### PR TITLE
[MOOV-1497]: Added models for PIS settings endpoint

### PR DESCRIPTION
**Pull request by agranillonf**
Added models for PIS settings endpoint

### CODE REVIEW

File: src/NoFrixion.MoneyMoov/Enums/BankAccountTypeEnum.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+//  Description: List of bank account types.
+// 
+//  Author(s):
+//  Axel Granillo (axel@nofrixion.com)
+// 
+//  History:
+//  08 03 2023  Axel Granillo   Created, Remote, Mexico City, Mexico.
+// 
+//  License:
+//  Proprietary NoFrixion.
```
Comment (sauravmaiti22) [inline src/NoFrixion.MoneyMoov/Enums/BankAccountTypeEnum.cs:13]: MIT license

File: src/NoFrixion.MoneyMoov/Models/Merchant/MerchantPisBankSetting.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+//  Description: Model representing an individual PIS bank setting.
+// 
+//  Author(s):
+//  Axel Granillo (axel@nofrixion.com)
+// 
+//  History:
+//  08 03 2023  Axel Granillo   Created, Remote, Mexico City, Mexico.
+// 
+//  License:
+//  Proprietary NoFrixion.
```
Comment (sauravmaiti22) [inline src/NoFrixion.MoneyMoov/Models/Merchant/MerchantPisBankSetting.cs:13]: MIT license

File: src/NoFrixion.MoneyMoov/Models/Merchant/MerchantPisBankSetting.cs [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace NoFrixion.MoneyMoov.Models;
+
+/// <summary>
+/// Represents an individual PIS bank setting.
+/// </summary>
+public class MerchantPisBankSetting
+{
```
Comment (sipsorcery) [inline src/NoFrixion.MoneyMoov/Models/Merchant/MerchantPisBankSetting.cs:26]: Maybe MerchantPayByBankSetting? While PIS makes sense to us it won't be immediately obvious to API users.

**Review by sipsorcery — CHANGES_REQUESTED** [review]

**Review by sauravmaiti22 — COMMENTED** [review]
Just change the license to MIT in all the new files.

**Review by sipsorcery — APPROVED** [review]

**Pull request merged**